pandas/_libs/index.pyx

```diff
@@ -98,6 +98,20 @@ cdef ndarray _get_bool_indexer(ndarray values, object val, ndarray mask = None):
     return indexer.view(bool)
 
 
+cdef _maybe_resize_array(ndarray values, loc, max_length):
```

Comment:
You will want to declare types for loc / max_length so it is clear how the comparisons are being done in the function body. I think Py_ssize_t is fine

Reply:
Updated, thanks